app/code/Magento/Catalog/Model/ProductRepository.php

```diff
@@ -534,6 +537,28 @@ public function save(\Magento\Catalog\Api\Data\ProductInterface $product, $saveO
         return $this->get($product->getSku(), false, $product->getStoreId());
     }
 
+    /**
+     * @param $product
+     */
+    protected function filterImageData($product)
```

Comment:
Please change protected to private due to Magento [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html)